parserator_web/templates/parserator_web/index.html

```diff
@@ -39,4 +39,58 @@ <h4>Parsing results</h4>
 
 {% block extra_js %}
   <script src="{% static 'js/index.js' %}"></script>
+  <script>
```

Comment:
this code looks good. could you move it to `js/index.js` instead of putting it inline here in the template?